lib/pbench/server/api/resources/query_apis/datasets/__init__.py

```diff
@@ -70,6 +70,7 @@ class RunIdBase(ElasticBase):
                 "sosreports",
             ],
         },
+        "tableOfContent": {"index": "run-toc", "whitelist": ["directory", "files"]},
```

Comment:
For consistency with the other keys, this should be all lowercase, and a single word. As I suggested earlier, I like "contents" as it's simple and descriptive. Something like "inventory" could also work, though I think that's less obvious.

Reply:
In the context of the whole URI, "contents" would probably be fine.  "inventory" would work, as would "files", I think.